docs/examples/multi-event.c

```diff
@@ -69,7 +69,7 @@ static void add_download(const char *url, int num)
   FILE *file;
   CURL *handle;
 
-  snprintf(filename, 50, "%d.download", num);
+  curl_msnprintf(filename, 50, "%d.download", num);
```

Comment:
Is there a reason to use `curl_msnprintf` in the examples? The docs for them discourage use in new applications

Reply:
It was already used in a couple places (on a quick glance: for not strong reason),
and I tried avoiding `snprintf()` that is banned elsewhere (and by default after this
patch), and being traditionally problematic with MSVC.

But, no issues allowlisting `snprintf` here and dropping existing curl printf uses
(and disallow them perhaps to keep it this way.)

I'll fix.